data/settings.js

```diff
@@ -10453,4 +10453,451 @@ If enabled, protocol-level SSL errors are logged. Same as
+server).
+
+Leave it commented out to bind anonymously (useful with
+\`passdb_ldap_bind = yes\`).
+
+Example: \`ldap_auth_dn = uid=dov-read,dc=ocn,dc=ad,dc=jp,dc=.\``
+	},
+
+	ldap_auth_dn_password: {
+		tags: [ 'ldap' ],
+		default: '<empty>',
+		values: setting_types.STRING,
+		text: `
+Password for LDAP server, used if [\`ldap_auth_dn\`](#dn) is specified.`
+	},
+
+	ldap_iterate_fields: {
+		tags: [ 'ldap' ],
+		default: '<empty>',
+		values: setting_types.STRLIST,
+		text: `
+::: info
+	LDAP Authentication Only
+:::
+
+Attributes to get a list of all users. Currently only the attribute
+\`user\` is supported.
+
+Example:
+\`\`\`
+	iterate_attrs {
+	user = %{ldap:mailRoutingAddress}
+	}
+\`\`\``
+	},
+
+	ldap_iterate_filter: {
+		tags: [ 'ldap' ],
+		default: '<empty>',
+		values: setting_types.STRING,
+		text: `
+::: info
+	LDAP Authentication Only
+:::
+
+Filter to get a list of all users.
+
+Example: \`ldap_iterate_filter = (objectClass=smiMessageRecipient)\``
+	},
+
+	ldap_version: {
+		tags: [ 'ldap' ],
+		default: '3',
+		values: setting_types.UINT,
+		text: `
+LDAP protocol version to use. Likely \`2\` or \`3\`.`
+	},
+
+	fields: {
+		tags: [ 'ldap' ],
+		default: '<empty>',
+		values: setting_types.STRLIST,
+		text: `
+::: info
+	LDAP Authentication Only
+:::
+
+Specify user attributes to be retrieved from LDAP in passdb look up.
+
+Password checking attributes:
+
+* \`user\`: Virtual user name (user@domain), if you wish to change the
+	user-given username to something else
+* \`password\`: Password, may optionally start with \`{type}\`, e.g., \`{crypt}\`
+
+Example:
+
+\`\`\`
+	fields {
+	password = %{ldap:userPassword}
+	user = %{ldap:mailRoutingAddress}
+	home = %{ldap:homeDirectory}
+	uid = %{ldap:uidNumber}
+	gid = %{ldap:gidNumber}
+	}
+\`\`\`
+
+There are also other special fields which can be returned. See
+[[link,passdb_extra_fields]].
+
+If you wish to avoid two LDAP lookups (passdb + userdb), you can use
+[[link,auth_prefetch]] instead of userdb ldap in \`dovecot.conf\`. In that
+case you'll also have to include \`user_attrs\` in \`fields\` field
+prefixed with \`userdb_\` string.
+FIXME: WHAT ABOUT THIS ?`
+	},
+
+	ldap_filter: {
+		tags: [ 'ldap' ],
+		default: '<empty>',
+		values: setting_types.STRING,
+		text: `
+::: info
+	LDAP Authentication Only
+:::
+
+Filter for password lookups (passdb lookup).
+
+Example: \`ldap_filter = (&(objectClass=posixAccount)(uid=%u))\``
+	},
+
+	ldap_auth_sasl_authz_id: {
+		tags: [ 'ldap' ],
+		default: '<empty>',
+		values: setting_types.STRING,
+		text: `
+SASL authorization ID, ie. the \`ldap_auth_dn_password\` is for this "master user", but the
+\`ldap_auth_dn\` is still the logged in user. Normally you want to keep this empty.`
+	},
+
+	ldap_auth_sasl_mechanism: {
+		tags: [ 'ldap' ],
+		default: '<empty>',
+		values: setting_types.STRING,
+		text: `
+SASL mechanism names (a space-separated list of candidate mechanisms) to use.`
```

Comment:
Oh, this is a list of multiple mechanisms? We should rename it to ldap_auth_sasl_mechanisms and make it a boollist. Is it in priority order also?